solr/bin/systemd/solr.service

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[Unit]
+Description=Apache Solr
+After=network.target
+
+[Service]
+Type=forking
+User=SOLR_USER
+Environment=SOLR_INCLUDE=/etc/default/SOLR_SERVICE.in.sh
+ExecStart=SOLR_INSTALL_DIR/bin/solr start
```

Comment:
There may be quotes needed if SOLR_INSTALL_DIR contains special characters

Reply:
Looks like some older Linux versions did not support quoted values here, but it should be ok now. I'll try.

Reply:
Hmm, the value substituted for `SOLR_INSTALL_DIR` is generated by `SAFE_SOLR_SERVICE=$(systemd-escape --path "$SOLR_SERVICE")` where `/foo bar/baz` is represented as `foo\x20bar-baz`, so I believe we're safe already.